emmy.go

```diff
@@ -133,12 +133,13 @@ func runClients(n int, concurrently bool, protocolType, protocolVariant, endpoin
 // Parameters passed to the client in client.ProtocolParams struct have fixed
 // values for demonstration purposes.
 func runClient(protocolType, protocolVariant, endpoint string) {
-	genClient := getProtocolClient(protocolType, protocolVariant, endpoint)
-	if genClient == nil {
-		cLogger.Critical("Error ocurred when creating client")
+	pbType, pbVariant, err := parseSchema(protocolType, protocolVariant)
+	if err != nil {
+		cLogger.Criticalf("%v", err)
 		return
 	}
 
+	c := client.NewClient(endpoint)
```

Comment:
Please see my comment above. Here you should be instantiating the target client directly and invoke its method afterwards.